Overview.src.html

```diff
@@ -908,6 +908,13 @@ <h4 id=terminology-headers>Headers</h4>
 period. How exactly a user agent can use "<code title>deprecated</code>" is not defined by
 this specification.
 
+<p>A <span title=concept-response>response</span> has an associated
+<dfn title=concept-response-policy-list>policy list</dfn>, which is a list of
+<a href=https://w3c.github.io/webappsec-csp/#policy>Content Security Policy objects</a>
+delivered in the <span title=concept-response>response</span>'s
+<span title=concept-response-header-list>header list</span>. The list is empty unless
+otherwise specified.
```

Comment:
Perhaps "CSP list" would be better? I suspect this might not be the last policy. Also, any particular reason the source is mentioned?